cmd/clusterctl/examples/openstack/provider-component/cluster-api/manager_image_patch.yaml

```diff
@@ -7,5 +7,5 @@ spec:
   template:
     spec:
       containers:
-      - image: gcr.io/k8s-cluster-api/cluster-api-controller:0.1.0
+      - image: gcr.io/k8s-cluster-api/cluster-api-controller:v0.1.4
```

Comment:
@gyliu513 here, the container was updated to 0.1.4, otherwise..we are still using 0.1.0 cluster-api

Reply:
Do we need to update https://github.com/kubernetes-sigs/cluster-api-provider-openstack/blob/e8da2d864b5d01c87d9f33b8b20acc1b4340fc51/Gopkg.toml#L62 ?

Reply:
no, this is branch... I actually already updated the code  to latest in the branch

Reply:
> I actually already updated the code to latest in the branch

Where did you update to latest for cluster API? Can you use tag here?

![image](https://user-images.githubusercontent.com/4461983/61769629-7a42e600-ae1d-11e9-8f86-0861bb95439f.png)

Reply:
ok, this might need some check ,I need double check whether it's feasible to do so
thanks for the reminder~

Reply:
I still did not found cluster api version was updated in Gopkg.toml?

Reply:
sorry, I mean, the PR are separated into 2 commits

I will update the commit message to make it more clear